### PR TITLE
Add skip condition if QT_API is no set (issue #1101)

### PR DESCRIPTION
**Pull request by jordi5**
`pytest tardis` fails on import of gui interface and widgets if QT_API variable is not set

## Description
 The issue appears on import (and I still don't how skipimport would work but I'll revisit at some point). I changed the call so that the import wouldn't check inside the module until needed for the test by importing the whole module. Following this some of the objects inside the function have to be called more explicitly.

I added the skipif decorator from pytest. The test now skips when the variable QT_API is not defined, but I'm not sure how the `pytest tardis` should proceed when the variable is defined. When I define QT_API either way on my master branch the test still skips, which I suppose is the expected behaviour because I didn't modify anything there. 

On a different commit I modified the functions interface and widgets to change how the Iphyton library calls specific functions, as can be seen this changes a qt4 to a qt5 and is purely for consistency, given that the functions are the same.

## Motivation and Context
Adresses #1101 

## How Has This Been Tested?
Run `pytest tardis` and the test now skips when QT_API is not declared.
Run again for the Iphyton calls change.
Giving( to save as reference)
263 passed, 1119 skipped, 1 xfailed in 24.38 seconds

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)

All input is appreciated.

### CODE REVIEW

**Comment by jordi5** [comment]
@wkerzendorf Could you please reopen this? 
I don't have permissions.
Thanks

**Comment by jordi5** [comment]
Ok, so this is ready to merge I think

File: tardis/gui/tests/test_gui.py [hunk 1]
```diff
@@ -3,9 +3,7 @@
 from tardis.io.config_reader import Configuration
 from tardis.simulation import Simulation
 import astropy.units as u
-from tardis.gui import interface
-from tardis.gui.widgets import Tardis 
-from tardis.gui.datahandler import SimpleTableModel
+import tardis.gui as gui
```
Comment (sashank27) [inline tardis/gui/tests/test_gui.py:6]:
```suggestion
from tardis.gui.widgets import Tardis 
from tardis.gui.datahandler import SimpleTableModel
```

File: tardis/gui/tests/test_gui.py [hunk 2]
```diff
@@ -39,11 +37,12 @@ def simulation_one_loop(
...
-
+    
+@pytest.mark.skipif('QT_API' not in os.environ, reason="enviroment variable QT_API is not set")
 def test_gui(simulation_one_loop):
     simulation = simulation_one_loop
     app = QtWidgets.QApplication([])
-    tablemodel = SimpleTableModel
-    win = Tardis(tablemodel)
+    tablemodel =gui.datahandler.SimpleTableModel
+    win = gui.widgets.Tardis(tablemodel)
```
Comment (sashank27) [inline tardis/gui/tests/test_gui.py:46]:
```suggestion
    tablemodel = SimpleTableModel
    win = Tardis(tablemodel)
```

File: tardis/gui/widgets.py [hunk 3]
```diff
@@ -1066,13 +1066,13 @@ def __init__(self, tablemodel, config=None, atom_data=None, parent=None):
         """
 
         #assumes that qt has already been initialized by starting IPython
-        #with the flag "--pylab=qt"
+        #with the flag "--pylab=qt"gut
```
Comment (jordi5) [inline tardis/gui/widgets.py:1069]: Fix this typo later

**Pull request merged**